CHANGES

```diff
@@ -28,6 +28,8 @@
 
 # Fix several static analysis and compiler warnings.
 
+# Add MPID_Pre_init to CH4 ADI.
```

Comment:
This is not specific to ch4. It is a new ADI function required for any device implementation.

Reply:
Also, we changed the function signature of `MPID_Init` because of this addition. We should mention that, too.

Reply:
Right. I'll update it.